sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionManager.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs;
+
+import com.azure.messaging.eventhubs.eventprocessor.models.Checkpoint;
+import com.azure.messaging.eventhubs.eventprocessor.models.PartitionOwnership;
+
+import java.util.List;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * Partition manager stores and retrieves partition ownership information and checkpoint
+ * details for each partition.
+ */
+public interface PartitionManager {
+
+    /**
+     * Called to get the list of all existing partition ownership from the underlying data store.
+     * Could return empty results if there are is no existing ownership information.
+     * @param eventHubName The event hub name
```

Comment:
nit: consistent blank spaces between parameters and documentation. You have them on a few up there and not on others. IntellIJ has a formatting setting for this.

Also, I noticed that the line wraps are inconsistent for your docs. IntelliJ also has a formatting setting for this. "Wrap at right margin" will correctly wrap your docs.
![image](https://user-images.githubusercontent.com/10136526/61977298-ef8bf380-afa2-11e9-9937-571592c33751.png)